installer/resources/pacbot_app/files/DB.sql

```diff
@@ -1708,7 +1708,8 @@ INSERT IGNORE INTO cf_Policy (`policyId`,`policyName`,`policyDesc`,`resolution`,
 INSERT IGNORE INTO cf_Policy (`policyId`,`policyName`,`policyDesc`,`resolution`,`policyUrl`,`policyVersion`,`status`,`userId`,`createdDate`,`modifiedDate`,`resolutionUrl`) VALUES ('Enable_pgAudit_Database_Flag_for_Postgresql_Instance','Enable cloudsql.enable_pgaudit Database Flag for Postgresql Instance','Ensure that the cloudsql.enable_pgaudit database flag is enabled for your Google Cloud PostgreSQL server instances in order to allow for database auditing that is often required to comply with government, financial, and ISO certifications.',NULL,NULL,'version-1',NULL,NULL,'2022-10-04','2022-10-04','https://github.com/PaladinCloud/CE/wiki/GCP-Policy#enable-cloudsqlenable_pgaudit-database-flag-for-postgresql-instance');
 
 INSERT IGNORE INTO cf_Policy (`policyId`,`policyName`,`policyDesc`,`resolution`,`policyUrl`,`policyVersion`,`status`,`userId`,`createdDate`,`modifiedDate`,`resolutionUrl`) VALUES ('Remove_user_options_for_SQLServer','Remove user options database flag for Cloud SQL SERVER instance','In order to avoid defining global defaults for all database users, delete user options database flag','1. Go to the Cloud SQL Instances page in the Google Cloud Console by visiting  https://console.cloud.google.com/sql/instances. \n2. Select the SQL server instance where the database flag needs to be enabled. \n3. Click Edit . \n4. Scroll down to the Flags section. \n5. Delete user options flag from database flags section \n6. Click Save . \n7. Confirm the changes under Flags on the Overview page.',NULL,'version-1',NULL,NULL,'2022-10-17','2022-10-17','https://github.com/PaladinCloud/CE/wiki/GCP-Policy#Remove-user-options-Database-Flag-for-SQL-server-Instance');
-INSERT IGNORE INTO cf_Policy (`policyId`,`policyName`,`policyDesc`,`resolution`,`policyUrl`,`policyVersion`,`status`,`userId`,`createdDate`,`modifiedDate`,`resolutionUrl`) VALUES ('Set_all_users_to_owner_email_notifications','Set All users with the following roles to owner in email notifications for MS defender.','In MS defender email alert settings, Ensure All users with following rules is set to owner.',NULL,NULL,'version-1',NULL,NULL,'2022-10-14','2022-10-14','https://github.com/PaladinCloud/CE/wiki/Azure-Policy#in-ms-defender-ensure-that-all-users-with-the-following-roles-is-set-to-owner');
```

Comment:
Please verify if this deletion is needed

Reply:
yes we have a duplicated record  for this policy.